src/main/webapp/js/views/question-views.js

```diff
@@ -821,6 +821,35 @@
 				
 				return rtn;
 			}});
+			
+			coll.add({key:"Sequence", value:function(textToProcess) {
+				// if the question is of type Sequence and textToProcess is not null string then divide the text up into 1..n tokens
+				// set the next sequence number to the count of choices
+				// add the choices and their sequence number to the collection as correct
+				// ignore the isCorrectBoxValue for Sequence questions
+				
+				var rtn = false;
+				
+				var cq = model_factory.get("currentQuestion");
+				var tokens = textToProcess.split('|');
+				var nextChoice = cq.attributes.choices.length;	//if length=1 next choice index is 1
```

Comment:
First, you should use `cq.get('choices')` instead of `cq.attributes.choices...` 

Second, be sure your variable names are explicit.. Choices are objects, so `nextChoice`, to me, at first glance, seems to imply this variable refers to a choice object, not an index.